### PR TITLE
12 tilkynning login/innskrá

### PR DESCRIPTION
**Pull request by tryggvi321**

### CODE REVIEW

**Comment by hinrikfp** [comment]
don't duplicate the models.py since it will create duplicate tables in the database and the data won't be shared

File: Project/Backend/signin_login_registration/models.py
Comment (hinrikfp) [inline Project/Backend/signin_login_registration/models.py:1]: delete this and just import home.models where you need it

File: Project/Backend/static/json/signins.json
Comment (hinrikfp) [inline Project/Backend/static/json/signins.json:1]: is this file still used?

**Comment by tryggvi321** [comment]
No the json file can be deleted

**Comment by tryggvi321** [comment]
Updated the files to use home models

**Pull request merged**